pyproject.toml

```diff
@@ -1,20 +1,27 @@
-[tool.poetry]
-name = "geoh5py"
-version = "0.12.0-alpha.1"
+[build-system]
+requires = ["poetry-core>=1.0.0", "setuptools"]
+build-backend = "poetry.core.masonry.api"
 
-license = "LGPL-3.0-or-later"
+[project]
+name = "geoh5py"
 description = "Python API for geoh5, an open file format for geoscientific data"
-authors = ["Mira Geoscience <support@mirageoscience.com>"]
+license = "LGPL-3.0-or-later"
+keywords = ["geology", "geophysics", "data", "interoperability"]
+version = "0.12.0a1"
+dynamic = ["dependencies", "classifiers", "requires-python"]
```

Comment:
according to https://python-poetry.org/docs/pyproject/#requires-python
there is no reason for making requires-python dynamic

Reply:
ok, changed it here: https://github.com/MiraGeoscience/geoh5py/pull/726/commits/41b249636d3a3630b2181b1b4a0b47d8bea63de9